nemo/core/classes/common.py

```diff
@@ -1114,3 +1125,26 @@ def disable_checks():
             yield
         finally:
             typecheck.set_typecheck_enabled(enabled=True)
+
+    @staticmethod
+    def set_semantic_check_enabled(enabled: bool = True):
```

Comment:
better to rename to `update_semantic_check` ?

Reply:
No. It mirrors the set_enabled() API